pdns/recursordist/RECURSOR-MIB.txt

```diff
@@ -60,6 +60,9 @@ rec MODULE-IDENTITY
     REVISION "202306080000Z"
     DESCRIPTION "Added metrics for NOD and UDR events"
 
+    REVISION "202311060000Z"
```

Comment:
```suggestion
    REVISION "202405230000Z"
```
It doesn't really matter, but it gives a rough idea of when the update was made :)